abci/cmd/abci-cli/abci-cli.go

```diff
@@ -333,6 +342,13 @@ func cmdTest(cmd *cobra.Command, args []string) error {
 				}, nil)
 			},
 			func() error { return servertest.Commit(ctx, client, []byte{0, 0, 0, 0, 0, 0, 0, 5}) },
+			func() error {
+				return servertest.PrepareProposal(ctx, client, [][]byte{
+					[]byte("abc"),
```

Comment:
why `abci` here?

Reply:
I have changed to a random transaction as this should technically pass the prepare proposal. But I have issues with these tests written the way they are.. These are app specific things and it is unclear whether any of these should pass or not by default.